src/property.h

```diff
@@ -71,15 +71,25 @@ struct Property : public detail::Property {
     PROPERTY(AUSSENTEMP, 0x000c, Type::et_dec_val);
     PROPERTY(SAMMLERISTTEMP, 0x000d, Type::et_dec_val);
     PROPERTY(SPEICHERISTTEMP, 0x000e, Type::et_dec_val);
+#if !defined(TTF_07_C)
     PROPERTY(VORLAUFISTTEMP, 0x000f, Type::et_dec_val);
+#endif
     PROPERTY(RAUMISTTEMP, 0x0011, Type::et_dec_val);
     PROPERTY(VERSTELLTE_RAUMSOLLTEMP, 0x0012, Type::et_dec_val);
     PROPERTY(EINSTELL_SPEICHERSOLLTEMP, 0x0013, Type::et_dec_val);
     PROPERTY(VERDAMPFERTEMP, 0x0014, Type::et_dec_val);
     PROPERTY(RUECKLAUFISTTEMP, 0x0016, Type::et_dec_val);
+#if defined(TTF_07_C)
+    PROPERTY(PROGRAMMSCHALTER, 0x005a);
```

Comment:
this is not working, yet :)

Can you comment it, until I figured it out to avoid confusion?